src/open_inwoner/accounts/tests/test_password_reset_view.py

```diff
@@ -4,8 +4,12 @@
 
 from open_inwoner.utils.test import ClearCachesMixin
 
+from freezegun import freeze_time
+
 
 class PasswordResetViewTests(ClearCachesMixin, WebTest):
+    # Freeze the time to ensure a consistent cache key for throttling
+    @freeze_time("2024-01-01T12:00:00")
```

Comment:
Looking at the implementation it makes sense why this was failing:

https://github.com/maykinmedia/open-inwoner/blob/develop/src/open_inwoner/utils/mixins.py#L54

If `time()` for instance is about to reach a value where `mod 60` equals zero, the cache key changes, meaning that the throttling essentially resets for that IP.

```python
>>> 1708335959 - (1708335959 % 60)
1708335900
>>> 1708335960 - (1708335960 % 60)
1708335960
```